<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayStack_gui</class>
 <widget class="QMainWindow" name="DisplayStack_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Stack Options</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="integrationTimeLine">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>300</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="integrationTimeLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Integration Time (s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lowerWavelengthCutoffLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Wavelength Cutoff (Å)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lowerWavelengthCutoffLine">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>3000</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="upperWavelengthCutoffLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8000</string>
    </property>
   </widget>
   <widget class="QLabel" name="upperWavelengthCutoffLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Wavelength Cutoff (Å)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wavelengthCalibrationBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Wavelength Calibration</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="flatCalibrationBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Flat Calibration</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hotPixelBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Hot Pixel Masking</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>-40</y>
      <width>20</width>
      <height>811</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="deadPixelBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Dead Pixels</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stackButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stack Images</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="timeAdjustmentBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Adjustment</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="obsList">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="obsTimeLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Observations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QListWidget" name="inputList">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files to process</string>
    </property>
   </widget>
   <widget class="QListWidget" name="wavelengthList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="flatList">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="sunsetList">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>81</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sunsetLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sunset Date</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>21</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loadStackButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load Stack</string>
    </property>
   </widget>
   <widget class="QListWidget" name="runList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="runLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Runs</string>
    </property>
   </widget>
   <widget class="QListWidget" name="targetList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>141</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="targetLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Targets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="targetButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load Target</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
